app/controllers/V1/failed_imports_controller.rb

```diff
@@ -0,0 +1,12 @@
+module V1
+  class FailedImportsController < ApplicationController
+    def create
+      failed_import = ::FailedImports::CreationService.call(params)
```

Comment:
I'm not convinced there needs to be an entire service for a `find_or_create_by` call and also why we're passing `id` in the first place. It should just be an autogenerated UUID that gets returned when the object is created

Reply:
I wanted there to be as much correlation between the app store records and the provider app records - purely to make it easier to perform any data cleaning/deleting processes between the services (as this is a table where the data is on both and needed to be on both)

Reply:
For example, we justify ensuring the ids match on Submissions between the services for a similar reason

Reply:
Oh you made the change already, my bad

Reply:
yeah agreed, am updating that, was referencing the 2nd part of the comment

Reply:
sorry I grouped the other comment in with this one too

Reply:
I also forgot about the UUID bit, I'll make another discussion for it